Dockerfile

```diff
@@ -15,19 +15,14 @@ RUN crudini --set /etc/ironic-inspector/inspector.conf DEFAULT auth_strategy noa
     crudini --set /etc/ironic-inspector/inspector.conf DEFAULT debug true && \
     crudini --set /etc/ironic-inspector/inspector.conf database connection sqlite:///var/lib/ironic-inspector/ironic-inspector.db && \
     crudini --set /etc/ironic-inspector/inspector.conf DEFAULT transport_url fake:// && \
-    crudini --set /etc/ironic-inspector/inspector.conf DEFAULT enable_mdns True && \
     crudini --set /etc/ironic-inspector/inspector.conf processing store_data database && \
     crudini --set /etc/ironic-inspector/inspector.conf processing ramdisk_logs_dir /shared/log/ironic-inspector/ramdisk && \
     crudini --set /etc/ironic-inspector/inspector.conf processing always_store_ramdisk_logs true && \
     crudini --set /etc/ironic-inspector/inspector.conf processing power_off false && \
     crudini --set /etc/ironic-inspector/inspector.conf pxe_filter driver noop && \
     crudini --set /etc/ironic-inspector/inspector.conf processing node_not_found_hook enroll && \
     crudini --set /etc/ironic-inspector/inspector.conf discovery enroll_node_driver ipmi && \
-    crudini --set /etc/ironic-inspector/inspector.conf processing processing_hooks "\$default_processing_hooks,extra_hardware,lldp_basic" && \
```

Comment:
Thinking about this one more - we can leave in extra_hardware and lldp_basic.  Is there any reason these need be removed for Stein?

Reply:
They're not removed completely, it's just the way github shows the diff (see line 25 below)

Reply:
yeah, this view can be confusing
the only change here is the removal of "&& \" at the end of the line

Reply:
That's what I get for reviewing late at night, thanks.